src/components/date-picker/pickers/MonthPicker.tsx

```diff
@@ -128,7 +128,6 @@ export class MonthPicker extends Picker {
         return (
             <div
                 className="datepicker-month"
-                // eslint-disable-next-line react/jsx-no-bind
```

Comment:
I don't know why these have worked until now and then suddenly stopped working when I removed `@stencil/eslint-plugin` 🤷‍♂️